pythonie/meetups/models.py

```diff
@@ -57,4 +58,5 @@ def __str__(self):
     @classmethod
     def future_events(cls):
         today = datetime.now()
-        return cls.objects.filter(time__gt=today).filter(time__lt=next_n_months(today, 3))
+        return cls.objects.filter(
+            time__gt=today).filter(time__lt=next_n_months(today, 3))
```

Comment:
HI @nicolasunravel , do you mean changing the filter somehow to avoid calling filter() twice?